src/main/protocol.ts

```diff
@@ -75,7 +82,10 @@ export const listenForProtocolHandler = () => {
     }
   });
 
-  if (process.platform === 'win32') {
+  // pass protocol URL via npm start args in dev mode
+  if (isDevMode() && process.env.npm_config_argv) {
+    scanNpmArgv(process.env.npm_config_argv);
+  } else if (process.platform === 'win32') {
```

Comment:
This probably shouldn't be an else if.

These conditions should both be able to happen at the same time

Reply:
Does this mean that if we're running `win32` and we're in dev mode, we want `handlePotentialProtocolLaunch` to run twice? It seems to me like duplicate code, but on the other hand, running `scanArgv` twice doesn't seem to have any negative side effects (tested on my end by just duplicating the `scanNpmArgv` function call).

Reply:
I think we do just want to run this once.